config/initializers/delayed_job_web.rb

```diff
@@ -0,0 +1,18 @@
+dj_enabled  = ENV['DELAYED_JOB_ADMIN_ENABLED']
```

Comment:
can this change to be

`dj_enabled = %w{ yes true 1 }.include?(ENV['DELAYED_JOB_ADMIN_ENABLED'])`

Just adds a bit of flexibility for if someones not sure which value to set

Reply:
👍🏽